website/docs/commands/publishing/publish.md

```diff
@@ -128,25 +131,37 @@ pass it like `--secret-key-password env:MY_KEY_PASSWORD`.
 
 Scala CLI can generate and keep a secret key for you. For that, create the key with
 ```sh
-scala-cli config --create-key
+scala-cli --power config --create-gpg-key
 ```
 
-You can then ask publish to use the key kept in the Scala CLI config with
-```sh
-scala-cli publish \
-  --secret-key config:pgp.secret-key \
-  --secret-key-password config:pgp.secret-key-password \
-  …
-```
+This generated key kept in config will be used by default unless specified otherwise, e.g.:
+- with directives:
+    ```bash
+    //> using publish.secretKey env:PGP_SECRET
+    //> using publish.secretKeyPassword command:get_my_password
+    ```
+
+- with options:
+    ```sh
+    scala-cli publish \
+      --secret-key env:PGP_SECRET \
+      --secret-key-password file:pgp_password.txt \
+      …
+    ```
+
+Since these values should be kept secret the options and directives accept the format documented [here](/docs/reference/password-options.md).
```

Comment:
```suggestion
Since these values should be kept secret, the options and directives accept the format documented [here](/docs/reference/password-options.md).
```